src/main/java/thePackmaster/cardmodifiers/strikepack/AddStrikeTagModifier.java

```diff
@@ -1,36 +1,36 @@
 package thePackmaster.cardmodifiers.strikepack;
 
 import basemod.abstracts.AbstractCardModifier;
+import com.badlogic.gdx.graphics.Color;
 import com.megacrit.cardcrawl.cards.AbstractCard;
 import com.megacrit.cardcrawl.core.CardCrawlGame;
 import com.megacrit.cardcrawl.dungeons.AbstractDungeon;
 import thePackmaster.SpireAnniversary5Mod;
 import thePackmaster.cardmodifiers.madsciencepack.AbstractMadScienceModifier;
 import thePackmaster.cardmodifiers.madsciencepack.AbstractMadScienceModifierWithName;
 
-public class AddStrikeTagModifier extends AbstractMadScienceModifierWithName {
+public class AddStrikeTagModifier extends AbstractCardModifier {
 
-    public AddStrikeTagModifier(int valueIn){
-        super(valueIn);
+    public AddStrikeTagModifier() {
+        super();
     }
 
     @Override
     public void onInitialApplication(AbstractCard card) {
-        super.onInitialApplication(card);
-        if (!card.hasTag(AbstractCard.CardTags.STRIKE)){
-            card.tags.add(AbstractCard.CardTags.STRIKE);
-            card.applyPowers();
-        }
+        card.tags.add(AbstractCard.CardTags.STRIKE);
```

Comment:
Why remove the check to not add a duplicate tag?

Reply:
The card itself already does the check, so this was an unnecessary double check.